example-app/src/main/java/com/adyen/checkout/example/data/storage/KeyValueStorage.kt

```diff
@@ -28,6 +28,8 @@ interface KeyValueStorage {
     fun isSplitCardFundingSources(): Boolean
     fun isAddressFormEnabled(): Int
     fun getInstantPaymentMethodType(): String
+    fun isInstallmentOptionsEnabled(): Int
```

Comment:
Can't we have one configuration for both sessions and non sessions?

Reply:
it's going to be a long list 
- no installment configuration & no session installments
- default installment configuration & no session installments
- default installment configuration & session installments
- default revolving installment configuration & no session installments
- default revolving installment configuration & session installments
- card based installment configuration  & no session installments
- card based installment configuration  & session installments

Reply:
How about the same 4 options that work for sessions and non sessions at the same time?